spec/services/handover_date_service_spec.rb

```diff
@@ -448,6 +495,50 @@
         end
       end
 
+      context 'when in a Womens prison' do
+        let(:prison) { womens_prison }
+        let(:arrival_date) { sentence_start_date }
+        let(:today) { sentence_start_date + 1.week }
+
+        context 'when before policy start date' do
+          let(:sentence_start_date) { womens_policy_start_date - 1.day }
+
+          it 'follows pre-policy rules' do
+            expect(pom).to be_supporting
+            expect(com).to be_responsible
+          end
+
+          context 'when transferred to open conditions (Cat T)' do
+            let(:category) { 'T' }
+
+            it 'follows pre-policy rules' do
+              expect(pom).to be_supporting
+              expect(com).to be_responsible
+            end
+          end
+        end
+
+        context 'when entering on or after the policy start date' do
+          let(:sentence_start_date) { womens_policy_start_date }
+
+          it 'follows POM policy rules' do
+            expect(pom).to be_responsible
+          end
+
+          it 'follows COM policy rules' do
+            expect(com).not_to be_supporting
```

Comment:
This might be a 'stronger' test:

```suggestion
            expect(com).not_to be_involved
```

i.e. asserting that the COM isn't involved whatsoever (rather than just 'not supporting')

I know that POM & COM can't both be responsible at the same time, but I don't think that's explicitly enforced unless we test for it.